FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/presenter/LoginPresenter.java

```diff
@@ -19,39 +19,46 @@ public LoginPresenter(LoginModel model, LoginView view) {
         this.view = view;
     }
 
-    @Subscribe
-    public void onFetchingFbUserDataCompletedEvent(FacebookLoginProvider.FetchingFbUserDataCompletedEvent event) {
+    public LoginModel getModel() {
```

Comment:
you dont need that. Its ony for private uses

Reply:
already deleted